zilencer/management/commands/populate_db.py

```diff
@@ -341,7 +341,8 @@ def send_messages(data):
     # type: (Tuple[int, Sequence[Sequence[int]], Mapping[str, Any], Callable[[str], Any]]) -> int
     (tot_messages, personals_pairs, options, output) = data
     random.seed(os.getpid())
-    texts = open("zilencer/management/commands/test_messages.txt", "r").readlines()
+    # texts = open("zilencer/management/commands/test_messages.txt", "r").readlines()
+    texts = open("zilencer/management/commands/new_test_data.txt", "r").readlines()
```

Comment:
I think we can just delete `test_messages.txt` and remove the commented-out version.

Also, I think rather than committing `new_test_data.txt`, we should instead just have code to check if it exists, and if it doesn't, run the generator script to create it.